doc/nrf/ug_zigbee_commissioning.rst

```diff
@@ -0,0 +1,214 @@
+The new device must be authenticated and authorized to become part of the network.
+For `Centralized security network`_, the authentication and authorization also involves establishing a unique Trust Center Link Key (TCLK).
+
+Network Formation
+=================
+
+The Network Formation mode is available only to Coordinators and Routers, because End Devices are not capable of forming a Zigbee network.
+
+.. figure:: /images/zigbee_commissioning_formation.svg
+   :alt: Zigbee Network Formation procedure
+
+   Zigbee Network Formation procedure
+
+In this mode, the node has not found a network to join during the Network Steering mode, and so it attempts to form a new network.
+Depending on the node role:
+
+* A Coordinator forms a centralised security network.
+* A Router forms a distributed security network.
+
+If Network Formation is successful, you can add nodes to the new network using `Network Steering`_.
+If unsuccessful, the commissioning procedure stops.
+
+Centralized security network
+----------------------------
+
+The centralized security network is started by the Zigbee Coordinator node, which assumes the Trust Center functionality.
+As such, it is responsible for managing the security and authorization in the network.
+In the centralized secutrity network, only the Trust Center can provide the network key to the joining device.
+This happens by using install code link keys specific to the joining device.
+The install code link key is exchanged with the Trust Center, which allows for encrypting packets exchanged between the Trust Center and the new node.
+This way, only the Trust Center and the new node can decrypt the packages exchanged between them.
+
+Distributed security network
+----------------------------
+
+The distributed security network is formed by Routers and does not have a centralized node that manages security in the network.
+In this network type, all Routers are equal and there is no root of trust between them.
+Each Router can provide the network key to the joining device.
+
+.. WIP
+
+   * Related to Touchlink -- routers set up an encrypted channel with other devices and share data
+
+Finding and Binding
+===================
+
+After the node joins a network, it can undergo the Finding and Binding procedure.
+With this procedure, you can establish connection between corresponding endpoints on two different nodes, which improves the communication speed between these nodes.
+This is because, once bound to each other, the nodes do not have to use the device address to communicate, but can rely on connections between the same clusters in their binding tables.
+
+.. figure:: /images/zigbee_commissioning_fb.svg
+   :alt: Zigbee Finding and Binding procedure
+
+   Zigbee Finding and Binding procedure
+
+Finding and Binding can be started automatically (using the ``bdbCommissioningMode`` mask) or manually.
+In this mode, the joining node is the target device, while the node already in the network is the initiator device.
```

Comment:
In the third clause, maybe it can be made more clear by adding "**that is**" like so:
`, while the node that is already in the network ...`